examples/boombox/index.html

```diff
@@ -24,6 +24,17 @@
 				transform: translate(-50%, -50%);
 				font-size: 20px;
 			}
+			#description {
+			    pointer-events: auto;
+			    font-family: sans-serif;
+			    padding: 1em;
+			    background-color:rgba(255,255,255,0.7);
+			    -webkit-backdrop-filter: blur(5px);
+			    position:absolute;
+			    bottom: 0px;
+			    left:0px;
```

Comment:
Should both 'left' and 'right' be defined?

Reply:
no, I don't think so.

Reply:
yes. I thought I did that everywhere. It ensures the div spans the whole screen.

Reply:
Hmmm. The MDN docs says that having both is an over-specification and only one is honored. Does it actually work?

Here's the MDN link: https://developer.mozilla.org/en-US/docs/Web/CSS/left

Reply:
Wow. That's so weird. I've never looked at the spec for left before.  Well, regardless of what the spec says, yet it works. When you absolutely position a div you can set the left and right but leave the width undefined, and the div will size itself based on the parent context.